src/layout/Loader/Loader.tsx

```diff
@@ -0,0 +1,18 @@
+import { Dna } from 'react-loader-spinner';
+import { Wrap } from './Loader.styled';
```

Comment:
the same - it's a perfect component for "components" folder (as I understand you use it for shared components)

Reply:
done